### DISCUSSION TITLE
I cannot see translations in tooltip

### DISCUSSION THREAD

**Discussion by aaykacc** [awcodes/filament-tiptap-editor#294] [Q&A]
hi everyone,

to begin with I want to thanks your to make this package.

I can' t see translations in tooltip. I published in vendor file

<img width="877" alt="Ekran Resmi 2023-12-31 22 14 38" src="https://github.com/awcodes/filament-tiptap-editor/assets/115133409/b36d4b8a-4a64-40cd-bda1-2f2658b5f9e1">

**Comment by awcodes** [comment]
For starters, you need to include the styles as described in the README.md. :)

But chances are you are using a language that the plugin doesn't have and your fallback language is not set up correctly in your app's config.

**Reply by MNhat0308** [reply]
so where i can override my own language?

**Reply by awcodes** [reply]
You can publish the translation files for the package and update or add what you need. It follows Laravel's standard usage of translation files.
